include/role/fileop.h

```diff
@@ -6,6 +6,13 @@ int librole_read_file_from_dir(const char * const directory,
     const char * const filename,
     struct librole_graph *role_graph);
 int librole_get_directory_files(const char * const directory, struct librole_graph *role_graph);
+int librole_get_system_roles(const char * const directory, char *system_roles[]);
```

Comment:
What's the point of moving functions to another header file?

Reply:
It needs for rolelst functioanlity. How to know about which role is system?